src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private final double side;
+
+    public Square(double side, String color) {
+        super(color);
+        this.side = side;
+    }
+
+    @Override
+    public double getArea() {
+        return side * side;
+    }
+
+    @Override
+    public void toDraw() {
+        System.out.println("Figure: square, area: " + getArea() + " sq"
+                + " side: " + side
+                + " color: " + color);
```

Comment:
The method name 'toDraw' does not follow the Java naming conventions for methods. Method names should be verbs and in mixed case with the first letter lowercase, with the first letter of each internal word capitalized. Consider renaming it to 'draw'.